.env-issuer.sample

```diff
@@ -12,8 +12,8 @@ ISSUER_API_AUTH_USER=user-issuer
 ISSUER_API_AUTH_PASSWORD=password-issuer
 ISSUER_KEY_STORE_ADDRESS=http://vault:8200
 ISSUER_KEY_STORE_PLUGIN_IDEN3_MOUNT_PATH=iden3
-ISSUER_REVERSE_HASH_SERVICE_URL=http://localhost:3001
-ISSUER_REVERSE_HASH_SERVICE_ENABLED=false
+
```

Comment:
Renaming this could be a breaking change for people using RHS. 

Anyway, I think nobody was using it on previous version. ?

Reply:
novody

Reply:
So.. no problem then. Ignore my comment